ls1intum/Artemis
# `Development`: Downgrade ng-bootstrap to fix an issue with modals

### Checklist
#### General



### Motivation and Context
Closes #6009 

### Description
- Reverted ng-bootstrap to v.13
- Broke a lot of tests which were fixed by upgrading ng-mocks which in turn broke some client tests who had mistakes like 'Imported ArtemisTestModule which exports mock of fa icon but also itself mock fa icon in the test itself' --> Fixed all the client tests

### Steps for Testing
Opening the tutorial group registration modal should not cause an error anymore (see screenshot). All other modals should also open without problems
![image](https://user-images.githubusercontent.com/29718932/209842519-c49a6b4e-153e-4d7f-b503-d35a70587bee.png)


### Review Progress

#### Code Review
#### Manual Tests
